### DISCUSSION TITLE
Version Warning Not Showing

### DISCUSSION THREAD

**Discussion by mdeliw** [squidfunk/mkdocs-material#3691] [Q&A] [answered]
First of all Thank you for this project. 

I am using mike and have followed the details outlined for Theme Extension using overrides. 

My project is at https://github.com/mdeliw/mdeliw.github.io/tree/gh-pages. I have checked everything, and missing something. Is this is insider feature only?

**Comment by squidfunk** [comment, marked as answer]
No, versioning was released some time ago as part of the community edition. There's likely a problem with your setup. Check out the [versioning example](https://github.com/squidfunk/mkdocs-material-example-versioning) – if you follow that closely, everything should work out.

**Reply by mdeliw** [reply]
This has been fixed. It was a timing issue with github. For some reason the version drop down was operating but the version warning wasn’t showing.
